#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/manpage-hyphen-fix.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Src/wmfrog.1: fix hyphen

@DPATCH@

diff --git a/Src/wmfrog.1 b/Src/wmfrog.1
index 36dc740..c0173e2 100644
--- a/Src/wmfrog.1
+++ b/Src/wmfrog.1
@@ -3,7 +3,7 @@
 .SH NAME
 wmfrog \- wmfrog
 .SH SYNOPSIS
-wmrog \fB\-s\fR <StationID> [-h] [-w <max wind>] [-t <time offset>] [-tmp <temp folder>] [-l <label>] [-m] [-delay <Time in Minutes>]
+wmrog \fB\\-s\fR <StationID> [\-h] [\-w <max wind>] [\-t <time offset>] [\-tmp <temp folder>] [\-l <label>] [\-m] [\-delay <Time in Minutes>]
 .SH DESCRIPTION
 \fB\-station\fR \fB\-s\fR <METAR StationID> 
 .IP
@@ -23,7 +23,7 @@ Set the maximum wind in KNOTS (for percentage of max / scaling.) default is 50 K
 .TP
 \fB\-o\fR <OFFSET>
 .IP
-Time offset of location from GMT (-8 for california, -1 for France etc...)
+Time offset of location from GMT (\-8 for california, \-1 for France etc...)
 .TP
 \fB\-tmp\fR <PATH>
 .IP
diff --git a/debian/control b/debian/control
index b97a75e..e268f27 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,8 @@ Section: x11
 Priority: optional
 Maintainer: Jari Aalto <jari.aalto@cante.net>
 Build-Depends: debhelper (>= 4.0.0), libxpm-dev
-Standards-Version: 3.6.2
+Standards-Version: 3.8.0
+Homepage: http://freshmeat.net/projects/wmfrog
 
 Package: wmfrog
 Architecture: any
@@ -16,4 +17,3 @@ Description: A dockapp for showing weather in graphical way
  perentage; Wind speed (average & gust); Wind Direction; Temperature
  (Celcius or Fareinheit); Station name can be configured and weather
  report time is displayed.
-
diff --git a/debian/copyright b/debian/copyright
index ba42872..dac4f6d 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,27 +1,39 @@
 This package was debianized by Jari Aalto <jari.aalto@cante.net> on
-Sat, 11 Feb 2006 15:16:55 +0200.
+2008-11-02 15:16:55+02 Sun.
 
-It was downloaded from http://www.colar.net/wmapps/
-See also http://dockapps.org/file.php/id/92
+It was downloaded from
 
-Copyright Holder: <tcolar@qarbon.com>
+    http://www.colar.net/wmapps
+
+    See also:
+    https://sourceforge.net/projects/wmfrog
+    http://freshmeat.net/projects/wmfrog
+    http://dockapps.org/file.php/id/92
+
+Copyright Holder:
+
+    <tcolar@qarbon.com>
 
 License:
 
-   This package is free software; you can redistribute it and/or modify
-   it under the terms of the GNU General Public License as published by
-   the Free Software Foundation; either version 2 of the License, or
-   (at your option) any later version.
+    This package is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
 
-   This package is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-   GNU General Public License for more details.
+    This package is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
 
-   You should have received a copy of the GNU General Public License
-   along with this package; if not, write to the Free Software
-   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+    You should have received a copy of the GNU General Public License
+    along with this package; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+
+The Debian packaging is licensed under the GPL v2 (or at your option,
+any later version), see below, and is:
+
+    Copyright (C) 2008 Jari Aalto <jari.aalto@cante.net>
 
 On Debian systems, the complete text of the GNU General
 Public License can be found in `/usr/share/common-licenses/GPL'.
-
diff --git a/debian/rules b/debian/rules
index fb32fd3..64ae934 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,41 +1,26 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# This file was originally written by Joey Hess and Craig Small.
-# As a special exception, when this file is copied by dh-make into a
-# dh-make output file, you may use that output file without restriction.
-# This special exception was added by Craig Small in version 0.37 of dh-make.
 
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-
-
-
-CFLAGS = -Wall -g
-
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
+MAKEDIR = Src
+MAKEOPT = -C $(MAKEDIR)
 
 configure: configure-stamp
+
 configure-stamp:
 	dh_testdir
-	# Add here commands to configure the package.
-
+	# Commands to configure the package.
 	touch configure-stamp
 
-
 build: build-stamp
 
-build-stamp: configure-stamp 
+build-stamp: configure-stamp
 	dh_testdir
 
-	# Add here commands to compile the package.
-	$(MAKE) -C Src
-	#docbook-to-man debian/wmfrog.sgml > wmfrog.1
+	# Commands to compile the package.
+	$(MAKE) $(MAKEOPT)
 
 	touch build-stamp
 
@@ -44,24 +29,24 @@ clean:
 	dh_testroot
 	rm -f build-stamp configure-stamp
 
-	# Add here commands to clean up after the build process.
-	-$(MAKE) -C Src clean
+	# Commands to clean up after the build process.
+	[ ! Src/Makefile ] || $(MAKE) $(MAKEOPT) clean
 
-	dh_clean 
+	dh_clean
 
 install: build
 	dh_testdir
 	dh_testroot
-	dh_clean -k 
+	dh_clean -k
 	dh_installdirs
 
 	# Add here commands to install the package into debian/wmfrog.
-	$(MAKE) -C Src install DESTDIR=$(CURDIR)/debian/wmfrog
+	$(MAKE) $(MAKEOPT) install DESTDIR=$(CURDIR)/debian/wmfrog
 
 
 # Build architecture-independent files here.
 binary-indep: build install
-# We have nothing to do by default.
+	# We have nothing to do.
 
 # Build architecture-dependent files here.
 binary-arch: build install
@@ -70,24 +55,24 @@ binary-arch: build install
 	dh_installchangelogs CHANGES
 	dh_installdocs
 	dh_installexamples
-#	dh_install
-#	dh_installmenu
-#	dh_installdebconf	
-#	dh_installlogrotate
-#	dh_installemacsen
-#	dh_installpam
-#	dh_installmime
-#	dh_installinit
-#	dh_installcron
-#	dh_installinfo
+#       dh_install
+#       dh_installmenu
+#       dh_installdebconf
+#       dh_installlogrotate
+#       dh_installemacsen
+#       dh_installpam
+#       dh_installmime
+#       dh_installinit
+#       dh_installcron
+#       dh_installinfo
 	dh_installman Src/wmfrog.1
 	dh_link
 	dh_strip
 	dh_compress
 	dh_fixperms
-#	dh_perl
-#	dh_python
-#	dh_makeshlibs
+#       dh_perl
+#       dh_python
+#       dh_makeshlibs
 	dh_installdeb
 	dh_shlibdeps
 	dh_gencontrol
